lib/DefineOpenCLWorkItemBuiltinsPass.cpp

```diff
@@ -194,13 +190,21 @@ bool DefineOpenCLWorkItemBuiltinsPass::defineGlobalIDBuiltin(Module &M) {
 
   auto GidBase = inBoundsDimensionOrDefaultValue(Builder, Dim, Result, 0);
 
-  // If we have a global offset we need to add it
   Value *Ret = GidBase;
-  if (clspv::Option::GlobalOffset()) {
-    auto Goff =
-        Builder.CreateCall(M.getFunction("_Z17get_global_offsetj"), Dim);
-    Goff->setCallingConv(CallingConv::SPIR_FUNC);
-    Ret = Builder.CreateAdd(GidBase, Goff);
+  if (clspv::Option::NonUniformNDRangeSupported()) {
+    auto Ptr = GetPushConstantPointer(BB, clspv::PushConstant::RegionOffset);
```

Comment:
The docs were a bit vague, is region offset the sum of global_offset and the clvk specific split of global ids? If not, should the offset still be added?

Reply:
Yes, it is. Maybe I could rename it to make this clearer (`global_plus_region_offset`?). Agree the docs need to say this explicitly.